app/views/foods/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<section class="my-blog bg-gradient">
+    <div class="container">
+		<div class="row add-food-item-action">
+			<div class="col-lg-6 col-md-8 add-food-item-container">
+				<div class="section-title text-center title-ex1">
+					<%= link_to 'Add Food Item', new_food_path, class:'btn btn-primary'%>
+				</div>
+			</div>
+		</div>
+		<div class="row">
+            <turbo-frame id="food-items-table-frame">
+                <table class="table" id="FoodItemsTable">
+                    <thead>
+                        <tr class="tableheader">
+                        <th class="tablecol sortable">Food Name</th>
+                        <th class="tablecol sortable">Measurement Unit</th>
+                        <th class="tablecol sortable">Price per Unit</th>
+                        <th class="tablecol sortable">Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <% @foods.each do |food| %>
+                            <tr class="tableheader">
+                                <td class="tablecol"><%= food.name %></td>
```

Comment:
You may include the missing methods in your foods controller
ie
```
def index
    @foods = Food.all # or any other logic to retrieve the list of foods
  end
```
The above is just an example to correct the below error

![Screenshot from 2024-02-08 18-59-23](https://github.com/Eccb7/fast-recipes/assets/100193711/347e031a-b648-4e3a-a07c-b0998cf8bdb0)